src/notebooks/controllers/commands/installPythonControllerCommands.ts

```diff
@@ -98,7 +98,7 @@ export class InstallPythonControllerCommands implements IExtensionSingleActivati
     }
 
     private async onDidChangeActiveNotebookEditor(editor: NotebookEditor | undefined) {
-        if (!this.isWeb && editor) {
+        if (!this.isWeb && editor && editor.notebook.notebookType === JupyterNotebookView) {
```

Comment:
Added a gate for the view here. If we fix context in the kernel picker commands this can be moved to the when, but that support is not there yet.